src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,70 @@
+/*eslint-disable*/
+import { FC } from 'react';
+import { Person } from '../types';
+import { useParams } from 'react-router-dom';
+import { PersoneLink } from './PersoneLink';
+export interface IPeopleTable {
+  people: Person[] | null;
+}
+export const PeopleTable: FC<IPeopleTable> = ({ people }) => {
+  const { slug } = useParams();
+
+  const allPeopleNames = people?.map(persone => persone.name);
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people?.map(person => {
+          return (
+            <tr
+              data-cy="person"
+              className={person.slug === slug ? 'has-background-warning' : ''}
```

Comment:
Use the classnames library for add class with condition

Reply:
Not fixed